ui/src/app/pages/analysis/leonardo-gce-launcher.spec.tsx

```diff
@@ -20,11 +20,11 @@ import userEvent from '@testing-library/user-event';
 import {
   genericProgressStrings,
   LeoApplicationType,
-  LeonardoAppLauncher,
+  LeonardoGCELauncher,
```

Comment:
renamed to avoid confusion!

This is a very old concept of "App" which is quite different from how we use it today.